internal/volumes/mount.go

```diff
@@ -134,7 +139,19 @@ func (s *LinuxMountService) Publish(targetPath string, devicePath string, opts M
 		return s.mounter.Mount(devicePath, targetPath, opts.FSType, mountOptions)
 	}
 
-	return s.mounter.FormatAndMount(devicePath, targetPath, opts.FSType, mountOptions)
+	formatOptions := make([]string, 0)
+
+	if opts.FsFormatOptions != "" {
+		lexer := shell.NewLex('\\')
+		formatOptions, err = lexer.ProcessWords(opts.FsFormatOptions, shell.EnvsFromSlice([]string{}))
+		if err != nil {
+			return err
+		}
+	} else if opts.FSType == "xfs" {
+		formatOptions = append(formatOptions, "-c", fmt.Sprintf("options=%s", XFSDefaultConfigPath))
+	}
```

Comment:
To make sure I understood this right: If the user specifies any options at all we pass them along. If they do not specify any options and use `xfs` we set the default config file.

If thats true, this behaviour should be added to the docs:

```
When using XFS as the filesystem type and no `fsFormatOptions` are set, we apply a default configuration to mkfs to ensure maximum compatibility with older Linux kernel versions. This configuration file is from the `xfs-utils` package and currently targets Linux 4.19.

> [!NOTE]
> The targeted minimum Linux Kernel version may be raised in a minor update, we will announce this in the Release Notes.

If you set any options at all, it is your responsible to make sure that all default flags from `mkfs.xfs` are supported on your current Linux Kernel version or that you set the flags appropriately.

```

Reply:
Yes this is correct. 

Added to docs :heavy_check_mark: